### PR TITLE
fix(tests): windows pathing

### PR DESCRIPTION
**Pull request by Sembauke**
Testing did not work on windows because of a Linux/Windows path issue.

```js
export function getProjectPath(): string {
  return (process.env.CALLING_DIR || process.cwd()) + path.sep;
}
```

returns `calling/dir\` on windows because the official separator on Windows seems to be ` "\" ` coming from `path.sep`

```js
export function getProjectName(): string {
  return getProjectPath().split(path.sep).slice(-2)[0];
}
```
thus having the string: ` "calling/dir\".split("\")` would return  an array `[calling/dir]`. This fails the test.

Windows works fine with using a front-slash and a back-slash to access any folder.

### CODE REVIEW

File: tools/challenge-helper-scripts/helpers/project-metadata.test.ts [hunk 1]
```diff
@@ -69,11 +69,17 @@ describe('getMetaData helper', () => {
   it('should throw if file is not found', () => {
     process.env.CALLING_DIR =
       'curriculum/challenges/english/superblock/mick-priject';
+
+    // determine if path is from Linux based or Windows based systems
+    const pathFromLinuxOrWin = path.sep.includes('/')
+      ? 'curriculum/challenges/_meta/mick-priject/meta.json'
+      : 'curriculum\\challenges\\_meta\\mick-priject\\meta.json';
```
Comment (nhcarrigan) [inline tools/challenge-helper-scripts/helpers/project-metadata.test.ts:76]: Could we not just use `path.join()`?

Comment (Sembauke) [reply tools/challenge-helper-scripts/helpers/project-metadata.test.ts:76]: Yes

**Review by naomi-lgbt — APPROVED** [review]
This should be good.

**Review by ShaunSHamilton — APPROVED** [review]
That took longer than expected.

Locally, the mobile curriculum tests still fail, but this does fix the mentioned issue. 👍

**Pull request merged**